test/options/rule-delimiter-scss/rule-delimiter.expected.scss

```diff
@@ -0,0 +1,19 @@
+.foo {
+  @import border-radius(5px);
+  background: red;
+
+.bar {
```

Comment:
Again this works via CLI, not sure why tests are different. Can only think its another rule handling it